.github/workflows/gke.yaml

```diff
@@ -0,0 +1,120 @@
+name: ConformanceGKE
+
+on:
+  pull_request_target:
+    types:
+      - "labeled"
+  #pull_request: #uncomment for testing
+  #  types:
+  #    - "labeled"
```

Comment:
Is this really possible for testing? Surely the point is that someone shouldn't be able to submit changes to this workflow and get it to run?

Reply:
Only possible if PR is based on `cilium/cilium` branch, like this PR: https://github.com/cilium/cilium/pull/15421

Reply:
The workflows run based on the events that exist in `master`. So unless it is merged uncommented on `master`, there will not be an issue => as André had told us during his initial presentation of GH Actions, it is the reviewers' duty to make sure no change to the workflows that allow this type of unsecured run happens.